Program.cs

```diff
@@ -1,143 +1,86 @@
-﻿// Welcome to AssemblyAI! Get started with the API by transcribing
-// a file using C#.
-//
-// In this example, we'll transcribe a local file. Get started by
-// downloading the snippet, then update the 'filename' variable
-// to point to the local path of the file you want to upload and
-// use the API to transcribe.
-//
-// IMPORTANT: Update line 130 to point to a local file.
-//
-// Have fun!
-
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
 using System.Text;
-using System.Threading;
+using System.Text.Json;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-public class FileUploader
-{
-    // Function to upload a local file to the AssemblyAI API
-    public async Task<string> UploadFileAsync(string apiToken, string path)
-    {
-        using var client = new HttpClient();
-        client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue(apiToken);
-
-        using var fileContent = new ByteArrayContent(File.ReadAllBytes(path));
-        fileContent.Headers.ContentType = new System.Net.Http.Headers.MediaTypeHeaderValue("application/octet-stream");
-
-        HttpResponseMessage response;
-        try
-        {
```

Comment:
No need for all the try/catches. If an exception is thrown, let it bubble up to the root and handle it there, or not. If an exception isn't caught, it should be logged to the console, so catching it yourself and then logging it to the console doesn't make much sense.